### ISSUE TITLE
Motion lost rtsp video signal but streaming still worked

### ISSUE THREAD

**Issue by mmallozzi** [Motion-Project/motion#265]
I set up Motion yesterday to capture stills periodically and on motion events from an RTSP IP webcam on my local network. It was working all through yesterday and overnight, saving many pictures successfully. My syslog shows that it hit a couple of events like:
```
netcam_interrupt_rtsp: Reading picture timed out for rtsp://...
netcam_handler_loop: Bad image.  Reconnecting with camera....
```

but it reconnected successfully. However, during the day today the same thing happened, but it failed to reconnect:
```
motion_loop: Video signal lost - Adding grey image
```

Thereafter it kept saving grey images according to my snapshot_interval=900 (every 15min). During this time however, I was able to connect to the RTSP stream successfully through VLC and through an RTSP streaming app on my phone, so it appears that the RTSP stream was fine and that Motion got into a state where it didn't try to reconnect. Does anyone know what might be going on?

I am running Ubuntu 14.04 and Motion version 3.4.1~rc01. I can create pastes of my motion.conf and /var/syslog if that would be helpful.

Thank you!

**Comment by Mr-DaveDev** [comment]
Some material improvements were made in the lost/reconnect processing that are included in version 4.0.1.  In the [releases](https://github.com/Motion-Project/motion/releases) tab you can find deb packages to upgrade from 3.4.1.

If this resolves it, please close issue.

**Comment by mmallozzi** [comment]
thank you! I'll install that now and reopen if it comes up again, since this was a one-time thing and I'll want to see it run for quite a while without issue before declaring it stable, so no need to leave this in the issue queue.
